### DISCUSSION TITLE
HD HOmeRUN Dual and Connect tuners

### DISCUSSION THREAD

**Discussion by TGsell1** [OpenSageTV/Community#149] [Tuners]
Is anyone interested in these tuners?   Free - just pay shipping.   Let me know via the forum and I will send e m address.  
Tom

**Comment by JustFred51** [comment]
I'm interested in both of them.  I'd like to add them to my setup for use with OpenDCT.  I'm one of the contributors to the SageTv and OpenDCT source code.   Any idea of cost for shipping to zip 94086?

**Comment by TGsell1** [comment]
Great! You’ve helped me numerous times.   I dont think GitHub will allow a complete mail address so try tomcgsell at g ….lSent from my iPhoneOn Aug 9, 2024, at 7:59 PM, Keith Fischer ***@***.***> wrote:﻿
I'm interested in both of them.  I'd like to add them to my setup for use with OpenDCT.  I'm one of the contributors to the SageTv and OpenDCT source code.

**Reply by JustFred51** [reply]
Email sent.

**Comment by LoisGNS** [comment]
@tgsell1, I know this is an older thread, but I'm wondering if you have any advise on getting OpenDCT working with HDHR Dual tuners? I have one, and although Sage sees it through OpenDCT, it won't tune any channels that way (tuner works ok when Sage sees it "native."

**Comment by TGsell1** [comment]
LoisGNS :  once open DCT recognized the ‘quad tuner’ i never went back to trying to use them as native.   I needed to use network encoders for my Cable Card which is now defunct hence OpenDCT.   I have found the quad tuner incredible stableAnd reliable.  Sent from my iPhoneOn Mar 4, 2026, at 2:35 AM, LoisGNS ***@***.***> wrote:﻿
@TGsell1, I know this is an older thread, but I'm wondering if you have any advise on getting OpenDCT working with HDHR Dual tuners? I have one, and although Sage sees it through OpenDCT, it won't tune any channels that way (tuner works ok when Sage sees it "native."
